packages/ui-components/src/components/MenuSection/MenuSection.component.tsx

```diff
@@ -18,8 +19,13 @@ const titleStyles = `
 	jn-text-xs
 	jn-p-2
 `
-/** Use MenuSection to structure and sub-divide MenuItems in a menu. All but the last MenuSection will render a visible divider at the bottom. Optionally, a MenuSection can have a title.*/
-export const MenuSection: React.FC<MenuSectionProps> = ({ title = "", children = null, className = "", ...props }) => {
+/** DEPRECATED: Use `PopupMenu` with `PopupMenu.Section` instead. Use MenuSection to structure and sub-divide MenuItems in a menu. All but the last MenuSection will render a visible divider at the bottom. Optionally, a MenuSection can have a title.*/
+const DeprecatedMenuSection: React.FC<MenuSectionProps> = ({
```

Comment:
(Question)
Would it possible to move this under a directory Deprecated?

Reply:
Let's do this in a separate PR for all deprecated components at once.